spec/controllers/submissions_controller_spec.rb

```diff
@@ -654,11 +654,11 @@
         grader_permission.save
       end
       context '#collect_submissions' do
-        before do
+        it('should respond with 204') do
           post_as grader, :collect_submissions,
                   params: { course_id: course.id, assignment_id: @assignment.id, groupings: [@grouping.id] }
+          expect(response.status).to eq 204
```

Comment:
Please modify the tests to use the newer `have_http_status` matcher, which produces a better error message.